OneCommunityGlobal/HighestGoodNetworkApp
# Bailey remove cropping from badges

# Description
(PRIORITY LOW) Jae: Remove the code cropping the badges.
Badges should show fully and not be cropped around the edges.

## Related PRS (if any):


## Main changes explained:
- Made changes to the Badge Component.
- Main changes were from the Badge.css file such as .badge_image_sm, md, and lg.
- These changes take place anywhere the Badges are displayed.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log in as either admin user, volunteer, or owner
4. go to dashboard→ Badges→ New Badges Earned→  Badges Earned Before Last Week→  Badge Report.
5. go to View Profile→ Featured Badges→ Assign Badges→   Select Featured


## Screenshots or videos of changes:

BEFORE CHANGES
![Badges BEFORE](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/108101478/5cebbe6c-38b0-404d-9cda-62c14e0df779)

AFTER CHANGES
![Badges AFTER](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/108101478/43ec4885-af0d-43a7-92fa-f0e07a9b8537)

![Badge Report Summary](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/108101478/b7e4f1aa-3c48-4109-871b-cd7ed47dba0e)

![Badge Image Medium](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/108101478/58941089-1fd0-4886-925e-acabfd6ce289)



## Note:
Badge Management functionality in other links is still being worked on for a other PR  so no badges will appear there right now.
